test/jpypetest/test_leak.py

```diff
@@ -135,30 +130,45 @@ def setUp(self):
                        # "-Xcheck:jni",
                        "-Xmx256M", "-Xms16M", classpath_arg)
 
+    def assertNotLeaky(self, function, counts=5000):
+        lc = LeakChecker()
+        assert not lc.memTest(function, counts), 'Potential leak found'
+
     @unittest.skipUnless(haveResource(), "resource not available")
     def testStringLeak(self):
         def stringFunc():
             jpype.java.lang.String('aaaaaaaaaaaaaaaaa')
-        runLeakChecker(stringFunc, 5000)
```

Comment:
These weren't actually asserting anything, so as a result wouldn't have failed if there was a leak. (I found this out when adding my own test)